float64 leg_pos_kp
float64 leg_pos_kd
float64 leg_for_kp
float64 leg_for_kd
float64 hip_pos_kp
float64 hip_pos_kd
float64 robot_ks
float64 robot_kt
float64 robot_kg
float64 left_toe_pos
float64 right_toe_pos
float64 left_leg_len
float64 right_leg_len
float64 left_leg_ang
float64 right_leg_ang
float64 slip_h
float64 slip_l0
float64 slip_m
float64 slip_k

bool hip_contr
bool leg_pos_contr
bool leg_slip_contr
bool left_leg_hop
bool right_leg_hop
bool two_leg_hop
bool alt_leg_hop
bool debug1
